### ISSUE TITLE
[UI] update status favicon while viewing logs

### ISSUE THREAD

**Issue by mitchhentges** [taskcluster/taskcluster#639]
When I'm viewing a task, the icon in the tab shows the status of the build (running, passed, failed) even if I change to a different tab.
It would be awesome if the status was still shown in the icon, even if I was viewing a log file [like this one](https://taskcluster-web.netlify.com/tasks/JU3ALKFIQTuN7kaTuTJ5KQ/runs/0/logs/live/https%3A%2F%2Fqueue.taskcluster.net%2Fv1%2Ftask%2FJU3ALKFIQTuN7kaTuTJ5KQ%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log)

**Comment by helfi92** [comment]
Agreed :)

**Comment by sonyshubham** [comment]
Hey,

Can I start working on this?
This will be my first contribution to Open Source, any guidance is appreciated.

**Comment by helfi92** [comment]
Go for it @sonyshubham. The ui [README](https://github.com/taskcluster/taskcluster/blob/master/ui/README.md) should help you get started. Let us know if you had any questions.

**Comment by helfi92** [comment]
How is it going @sonyshubham? Let me know if you had questions for me.

**Comment by Shubhamchinda** [comment]
Hey, @helfi92 , Can I take this issue?

**Comment by helfi92** [comment]
Go for it.

**Comment by Shubhamchinda** [comment]
Thanks, could you please tell more details about the issue?

**Comment by helfi92** [comment]
If you navigate to the Task view of a task (e.g., https://taskcluster-ui.herokuapp.com/tasks/LoVU_7KCSGK0pKI92GNTCQ), you will be able to see the current status in the favicon:

![Screen Shot 2019-07-19 at 8 03 35 AM](https://user-images.githubusercontent.com/3766511/61533750-bfb97a80-a9fb-11e9-973b-5a47ef8f5951.png)

Clicking on the "View Live Log" button of that view will bring you to https://taskcluster-ui.herokuapp.com/tasks/CZsHuuVNSWO0Gm6DZqw75A/runs/0/logs/https%3A%2F%2Fqueue.taskcluster.net%2Fv1%2Ftask%2FCZsHuuVNSWO0Gm6DZqw75A%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log. There's currently no favicon set in that view:

![Screen Shot 2019-07-19 at 8 04 49 AM](https://user-images.githubusercontent.com/3766511/61533831-eecfec00-a9fb-11e9-8ec4-b081b86147a0.png)

In other words, the user is not able to see what is the status of the task log they're viewing. We need to make a change to have the task status show up in the favicon. The code changes will be in the `ui` directory of this monorepo.

**Comment by Shubhamchinda** [comment]
@helfi92 Thanks for the details, So we need to show the same status of the task in the "task log" favicon?

**Comment by helfi92** [comment]
Yeah.

**Comment by Shubhamchinda** [comment]
> Yeah.

Hey @helfi92 

![Screenshot (195)](https://user-images.githubusercontent.com/21038543/61577488-82d9aa80-ab05-11e9-96bc-b215d255b57c.png)

Is this the fix you're looking for?

**Comment by globefire** [comment]
@helfi92 is this issue assigned? I would love to contribute to this issue.

**Comment by helfi92** [comment]
> Is this the fix you're looking for?

Yeah.

**Comment by helfi92** [comment]
> @helfi92 is this issue assigned?

Yeah. It's assigned to @Shubhamchinda.

**Comment by sonyshubham** [comment]
> How is it going @sonyshubham? Let me know if you had questions for me.

@helfi92 sorry for the delayed response, was travelling and did not have access to internet. Since it is assigned to @Shubhamchinda now is there anything else I can work on. 

Thanks

**Comment by helfi92** [comment]
No worries. Let me know if there an unassigned issue that you're interested in tackling. We also have many issues that are tracked in bugzilla. You can look at https://codetribute.mozilla.org/projects/taskcluster.

**Comment by sonyshubham** [comment]
> No worries. Let me know if there an unassigned issue that you're interested in tackling. We also have many issues that are tracked in bugzilla. You can look at https://codetribute.mozilla.org/projects/taskcluster.

Thanks for the tip, will check out the issues and proceed accordingly.
